HttplugBundle.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace Http\ClientBundle;
+
+use Symfony\Component\HttpKernel\Bundle\Bundle;
+
+/**
+ * @author David Buchmann <mail@davidbu.ch>
+ */
+class HttplugBundle extends Bundle
+{
```

Comment:
Not sure what is the PSR2 standard here: empty line between curly braces necessary or not? @hannesvdvreken you corrected me earlier.